sematic/config/tests/test_settings.py

```diff
@@ -257,23 +253,3 @@ def test_env_override_absent_plugin_no_file(no_settings_file):
 def test_get_active_settings_no_file(no_settings_file):
     active_settings = get_active_settings()
     assert active_settings == EXPECTED_DEFAULT_ACTIVE_SETTINGS
-
-
-def test_get_active_settings_env_scope_override():
```

Comment:
Does this test still pass after your changes? If so: why remove it? If not: I think it should

Reply:
Context: This test was actually reading from the local settings file, as no fixture for the settings file was specified.

This test doesn't pass after these changes if you local settings file does not contain `RAY_SUPPORTS_GPUS`, because the settings returned by `get_active_settings` are not overridden from the env for `RAY_SUPPORTS_GPUS`.

The setting is still overridden when you explicitly ask for it. So this would pass:
```
    with environment_variables(
        {
            PluginScope.EXTERNAL_RESOURCE.value: scope_env_var,
            StandardKuberaySettingsVar.RAY_SUPPORTS_GPUS.value: "true",
        }
    ):
        get_active_settings()
        assert get_plugin_setting(StandardKuberayWrapper, StandardKuberaySettingsVar.RAY_SUPPORTS_GPUS) == "true"
```

But this exact test case is covered by `test_env_override_specific_plugin_no_file` just above.

Reply:
Cool, thanks for the explanation! I also manually confirmed that the Kuberay plugin GPU configuration is still working via helm chart.